### DISCUSSION TITLE
Atmosphere table choices

### DISCUSSION THREAD

**Discussion by danangsetyo5364** [phoebe-project/phoebe2#852] [Questions]
Hallo! Sorry I just want to ask a quick question... When we installed the phoebe via `pip` command. Would all the atmosphere table such as `ck2004`, `phoenix`, be install automatically? Because when I want to compute a light curve, this message is always appear. Any idea about this?

my system teff primary = 7840K and Teffratio ~0.5

![Screenshot 2024-04-22 141018](https://github.com/phoebe-project/phoebe2/assets/108653905/caee0fb0-f023-4f22-830c-d7a88ce1cd9f)

Thanks!

**Comment by Raindogjones** [comment]
Hi, yes, the passband files contain ck2004 and blackbody by default and then automagically update to include phoenix and extinction tables when requested.  Your error is because at least part of one of the stars is out of the range of ck2004 atmosphere models, perhaps in surface gravity (difficult to tell without more details of your binary) or in temperature (the secondary is close to the limit, so there might be part of the star that falls off due to gravity darkening).  If the latter is the problem, you might be able to get around it by setting the secondary (or even both) to use phoenix atmospheres.
